volumina/layerstack.py

```diff
@@ -312,3 +312,11 @@ def _onSelectionChanged(self, selected, deselected):
             self[idx.row()].setActive(False)
         for idx in selected.indexes():
             self[idx.row()].setActive(True)
+
+    def toggleTopToBottom(self, target: Layer) -> None:
+        index = self.layerIndex(target)
```

Comment:
Since it's somewhat hard to convey everything in the method's name, a brief docstring would be nice to have.

```suggestion
    def toggleTopToBottom(self, layer: Layer) -> None:
        """Select layer and move it to the top or bottom of the stack.

        Layer goes to the top unless it's already there.
        Top layer goes to the bottom.        
        """
        index = self.layerIndex(layer)
```